src/engine/V3Eval.v3

```diff
@@ -1060,6 +1061,28 @@ def do_v_v_x8_pairwise_ext_x4(a: (u64, u64), f: (u16, u16) -> u32) -> (u64, u64)
 	}
     return (low, high);
 }
+def do_vv_v_dotprod<Tn, Tw, Uw>(a: (u64, u64), b: (u64, u64), dst_bits: byte, view: u64 -> Tn, extend: Tn -> Tw, convert: Tw -> Uw,
```

Comment:
I appreciate the effort that you put into making the generic routine, but until there are two or more examples, let's keep it simple and have just a routine that implements the one required instruction.

Reply:
No problem. It's a simple revert.